west.yml

```diff
@@ -266,7 +266,7 @@ manifest:
       groups:
         - hal
     - name: liblc3
-      revision: 1a5938ebaca4f13fe79ce074f5dee079783aa29f
+      revision: bb85f7dde4195bfc0fca9e9c7c2eed0f8694203c
```

Comment:
Isn't this version 1.1.1? Your commit message says 1.1.0

Reply:
Indeed, I took version from https://github.com/google/liblc3/blob/main/meson.build. Must be typo there